app/src/main/java/org/simple/clinic/teleconsultlog/success/TeleConsultSuccessUiRenderer.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.teleconsultlog.success
+
+import org.simple.clinic.mobius.ViewRenderer
+
+class TeleConsultSuccessUiRenderer(
+    private val ui: TeleConsultSuccessUi
+) : ViewRenderer<TeleConsultSuccessModel> {
+  override fun render(model: TeleConsultSuccessModel) {
+    ui.showPatientInfo(model.patient!!)
```

Comment:
We'll need to add a check here if the patient has not loaded, otherwise the app will crash on initial load.